lib/chartmogul/contact.rb

```diff
@@ -34,6 +34,7 @@ def self.all(options = {})
 
     def self.merge!(into_uuid:, from_uuid:)
       custom!(:post, "/v1/contacts/#{into_uuid}/merge/#{from_uuid}")
+      true
```

Comment:
Should the returned value be dependent on the result of `custom!` response?

Reply:
From my testing it either raised an error, or returned a HTTP 201 with an empty body. This is also how other `.merge` methods were defined before my changes, so I kept parity with them.

Reply:
Without this return what will happen is this:

```ruby
    65:   a = custom!(:post, '/v1/customers/merges', options)
 => 66:   binding.pry
    67:   true
    68: end

[1] pry(ChartMogul::Customer)> a
=> #<ChartMogul::Customer:0x00000001035085d0>
```

So it's not really useful, we try to parse the empty body, nothing is there, we create a new instance of `Customer` with no data in it.